docker-compose.yml

```diff
@@ -53,7 +53,7 @@ services:
     volumes:
       - ./docker/postgres/docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d
       - ./docker/postgres/docker-entrypoint-preinitdb.d:/docker-entrypoint-preinitdb.d
-      # - ./docker/postgres/persistence:/bitnami/postgresql
+      - ./docker/postgres/persistence:/bitnami/postgresql
```

Comment:
I'd leave this part commented out, since I don't think it's generally useful/desired behavior to have DB state persist between tear-downs. As long as we think it will work if someone chooses to uncomment it locally, I think we're good.